Frameworks/AudioToolbox/AudioFile.mm

```diff
@@ -580,11 +622,14 @@ DWORD AudioFileOpenWithCallbacks(void* context,
                                  AudioFile_SetSizeProc setSizeFunc,
                                  DWORD typeHint,
                                  AudioFileID _Nullable* out) {
-    EbrCallbackFile* callbackFP = new EbrCallbackFile(context, readFunc, writeFunc, getSizeFunc, setSizeFunc);
-    EbrFile* in = EbrAllocFile(callbackFP);
-
-    char header[4] = { 0 };
-    EbrFread(&header, sizeof(header), 1, in);
+    NSInputStream* in = [NSCallbackInputStream inputStreamWithCallBacks:context
+                                                               readFunc:readFunc
+                                                              writeFunc:writeFunc
+                                                            getSizeFunc:getSizeFunc
+                                                            setSizeFunc:setSizeFunc];
+    [in open];
+    char header[4] = {};
```

Comment:
hilariously no because of the use of strncmp

---
In reply to: [83959944](https://github.com/Microsoft/WinObjC/pull/1175#discussion_r83959944) [](ancestors = 83959944)